### PR TITLE
usertx accumulated fees revert

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
add a mapping between originaltx and usertx for relayed tx to help with reverting fees

### CODE REVIEW

File: process/block/postprocess/feeHandler.go [hunk 1]
```diff
@@ -69,6 +72,54 @@ func (f *feeHandler) GetDeveloperFees() *big.Int {
 
 // ProcessTransactionFee adds the tx cost to the accumulated amount
 func (f *feeHandler) ProcessTransactionFee(cost *big.Int, devFee *big.Int, txHash []byte) {
+	f.mut.Lock()
```
Comment (iulianpascalau) [inline process/block/postprocess/feeHandler.go:75]: :+1: 
looks good & clean the new implementation

File: process/transaction/shardProcess.go [hunk 2]
```diff
@@ -494,7 +495,11 @@ func (txProc *txProcessor) processMoveBalance(
 		return err
 	}
 
-	txProc.txFeeHandler.ProcessTransactionFee(moveBalanceCost, big.NewInt(0), txHash)
+	if isUserTxOfRelayed {
```
Comment (iulianpascalau) [inline process/transaction/shardProcess.go:498]: one more if in processing :D
Do not have a brighter idea though

**Review by iulianpascalau — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]
Looks good!

**Review by popenta — APPROVED** [review]
@@                            Log scanner                                  @@
##                            usertx-accumulated-fees-revert-logs          ##
================================================================================
- Known Warnings              623
- New Warnings                562
- Known Errors                0
- New Errors                  0
- Panics                      0
================================================================================

**Pull request merged**